src/tickets/TicketService.ts

```diff
@@ -0,0 +1,59 @@
+import { ITicket, TicketStatus } from "@app/entities/ITicket";
+import { TicketRepository } from "@app/tickets/TicketRepository";
+import { TicketResponse } from "@app/tickets/dto/TicketResponse";
+import _ from "lodash";
+import { HttpError } from "routing-controllers";
+import { Service } from "typedi";
+@Service()
+export class TicketService {
+  async getAllTickets() {
+    const repo = new TicketRepository();
+    const lst = await repo.list();
+    const new_lst = lst.map((each) => TicketResponse.toResponse(each));
+    return new_lst;
+  }
```

Comment:
the variable should be camel case
```suggestion
    const tickets = await repo.list();
    const results = lst.map((each) => TicketResponse.toResponse(each));
    return results;
  }
```